components/home/Investors.vue

```diff
@@ -0,0 +1,341 @@
+<template>
+  <section>
+    <v-container>
+      <v-stack justify="space-between" md-direction="column" align="start">
+        <v-stack direction="column">
+          <app-section-descriptor
+            title="our investors"
+            heading="They’re Awesome Investors. And They Chose To Back Us!"
+            style="max-width: 40rem; margin-bottom: 1.25rem"
+          />
+          <app-section-descriptor
+            description="Everyone of these great investors are well known and well respected in their own right. Through us, they’re helping shape the world like the way they envision it should be. "
+            style="max-width: 60rem"
+          />
+        </v-stack>
+        <v-button
+          type="link"
+          label="View all"
+          label-transform="uppercase"
+          style="margin-top: 2rem"
+        />
+      </v-stack>
+    </v-container>
+    <div
+      class="position-relative investors"
+      style="margin: 3rem auto 0; max-width: 1920px"
+    >
+      <div style="max-height: 540px; height: 50vh" />
+      <div
+        v-for="investor in investors"
+        :key="investor.key"
+        :class="['position-absolute', investor.cssClass]"
+      >
+        <v-image
+          :src="investor.image"
+          :title="investor.name"
+          :alt="investor.name"
+          @click.stop="showDesc(investor.key)"
+        />
+        <v-image
+          v-if="investor.orb"
+          :src="`/images/orb-${investor.orb.color}.svg`"
+          class="position-absolute"
+          :style="investor.orb.style"
+        />
+      </div>
+      <div v-if="investorDescription" class="investor-description-container">
+        <v-stack direction="column" align="center">
+          <v-image
+            src="/images/investor-close-btn.svg"
+            class="investor-description-close-btn"
+            @click.stop="hideDesc()"
+          />
+          <div class="investor-description">
+            <v-stack align="center">
+              <v-image
+                src="/images/investor-placeholder.png"
+                class="mobile-remove"
+              />
+              <v-stack direction="column" style="padding: 2rem">
+                <v-heading level="2" class="investor-name">
+                  {{ selectedInvestor.name }}
+                </v-heading>
+                <v-text>{{ selectedInvestor.desc }}</v-text>
+              </v-stack>
+            </v-stack>
+          </div>
+        </v-stack>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import AppSectionDescriptor from '../AppSectionDescriptor.vue'
+import VButton from '../lib/VButton.vue'
+import VContainer from '../lib/VContainer.vue'
+import VHeading from '../lib/VHeading.vue'
+import VImage from '../lib/VImage.vue'
+import VStack from '../lib/VStack.vue'
+import VText from '../lib/VText.vue'
```

Comment:
Please use root relative imports for internal modules.

```
import AppSectionDescriptor from '~/components/AppSectionDescriptor`
```

Reply:
@punaifang created root relative imports for internal modules